modules/test/playwright/tests/export-import-web/main/instance.export.spec.ts

```diff
@@ -20,21 +25,241 @@ import performLogin, {
 import {getTempDir} from '../../../utils/temp';
 import {readFileFromZip} from '../../../utils/zip';
 import {companyExportImportPageTest} from './fixtures/companyExportImportPagesTest';
+import {exportImportPagesTest} from './fixtures/exportImportPagesTest';
 import {toDateRangeDate, toDateRangeTime} from './utils/dateRangeUtil';
 import {objectDefitionRequestData} from './utils/objectDefitionRequestData';
 
 export const test = mergeTests(
 	applicationsMenuPageTest,
 	companyExportImportPageTest,
 	dataApiHelpersTest,
+	exportImportPagesTest,
 	featureFlagsTest({
 		'LPD-35914': {enabled: true},
 	}),
 	loginTest(),
+	objectPagesTest,
 	productMenuPageTest,
 	uiElementsPageTest
 );
 
+const rootModelTest = mergeTests(
+	test,
+	featureFlagsTest({
+		'LPD-34594': {enabled: true},
+		'LPD-35914': {enabled: true},
+	})
+);
+
+rootModelTest.describe(
+	'Manage export and import of root model object definitions',
+	() => {
+		rootModelTest(
+			'cannot enable inheritance from multiple parents when there are already child entries.',
```

Comment:
Hi @larissacribeiro, could you double check the test name? I don't see a direct relationship between the test names and the steps.